### ISSUE TITLE
rtsp url 

### ISSUE THREAD

**Issue by pradeepp159817** [geeksville/Micro-RTSP#18]
what is the rtsp url i should use in order to connect to rtsp server

**Comment by sowbug** [comment]
`rtsp://[my esp32's ip address]:8554/mjpeg/1`

This worked in VLC.

**Comment by pradeepp159817** [comment]
yes for me too but unable to work in android ...can u help with Android



-- 

Thanks and Regards
K . Pradeep varma
Co-founder, Kshemin

phone no: +91-7981186405

**Comment by sowbug** [comment]
Works for me on Android. TinyCam Pro. Same URL and option "RTSP over TCP (H264/H265)". Good luck.

**Comment by pradeepp159817** [comment]
Thanks, I'll check it out.



-- 

Thanks and Regards
K . Pradeep varma
Co-founder, Kshemin

phone no: +91-7981186405

**Comment by ewrfedf** [comment]
> Works for me on Android. TinyCam Pro. Same URL and option "RTSP over TCP (H264/H265)". Good luck.

Yes，That really helped me，and I want to visite the camera from remote net，how can I do，thx again.
